tutorials/2_sprite_animations_gestures/README.md

```diff
@@ -0,0 +1,241 @@
+# Basic: Sprites, Animations and Gestures
+
+This tutorial will introduce you to:
+
+ - `Sprite`: Sprites are how we draw images, or portions of an image in Flame.
+ - `SpriteAnimation`: SpriteAnimations are animations composed from sprites, where each sprite represents a frame.
+ - Gesture input: Input is a crucial part of game development, on this tutorial you will learn how to use one of the most basic of them, the tap detector.
+
+All the code of this tutorial code can be found [here](./code) and is based on the Flame `1.0.0-rc8` version.
+
+By the end of this tutorial you will have built a simple game which renders a button which when pressed, makes a small vampire robot run, that will look like this:
+
+![Preview](./media/preview.gif)
+
+## Sprite
+
+Before starting coding our game, it is important to understand what sprites are and what they are used for.
+
+Sprites are images, or a portion of an image loaded into the memory, which can then be used to render graphics on your game canvas.
+
+Sprites can be (and usually are) bundled into single images, that is a very useful technique as it lowers the amount of I/O operations needed to load the game assets because it is faster to load 1 image of 10k than to load 10 images of 1kb each.
+
+For example, on this tutorial, we will have a button that makes our robot run, this button needs two sprites, one for the unpressed button and another one for the pressed state, so we can have the following image:
+
+![Sprite example](code/assets/images/buttons.png)
+
+This technique is often called sprite sheet.
+
+## Animations
+
+Animations is what gives 2D games life, Flame provides a handy class called `SpriteAnimation` which lets you create an animation out of an list of sprites. Animations usually have all its frame's sprite into a single image
+
+![Animation example](code/assets/images/running.png)
+
+Flame provides way for easily turning that sprite sheet into an animation, which we will see how in a few moments.
```

Comment:
```suggestion
Flame provides a way for easily turning that sprite sheet into an animation, which we will see how in a few moments.
```

also we should probably decide on a standard way of saying `sprite sheet` (no opinions here)